guides/common/modules/proc_uninstalling-openscap-plugin.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Uninstalling_OpenSCAP_Plugin_{context}"]
+= Uninstalling OpenSCAP Plugin
+
+If you have previously installed the OpenSCAP plug-in but do not use it anymore to scan managed hosts, you can uninstall it from your {ProjectServer}.
+
+.Procedure
+. Uninstall the OpenSCAP plug-in from your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --no-enable-foreman-cli-openscap \
+--no-enable-foreman-plugin-openscap \
+--no-enable-foreman-proxy-plugin-openscap
```

Comment:
Technically this disables management in the installer, but that simply means it's unmanaged. It remains installed and active.